airsonic-main/src/main/java/org/airsonic/player/ajax/PlayQueueInfo.java

```diff
@@ -33,17 +33,19 @@
     private final List<Entry> entries;
     private final boolean stopEnabled;
     private final boolean repeatEnabled;
-    private final boolean radioEnabled;
+    private final boolean shuffleRadioEnabled;
+    private final boolean internetRadioEnabled;
```

Comment:
I'm not hugely familiar with the radio feature as is so maybe I'm just naiive, but what is the difference between internetRadio and shuffleRadio?

Reply:
Shuffle Radio is what you get when you click on More > Shuffle Radio, kind of like a weak smart playlist with a few useful filters. Internet Radios are basically external playlists.

Reply:
(The two are mutually exclusive though, it might actually be a good idea to merge them in some kind of "mode" variable)

Reply:
Okay them being mutually exclusive is the part I was trying to understand and now I understand the need for two different variables.